poc_iot_verifier/src/runner.rs

```diff
@@ -233,8 +233,7 @@ impl Runner {
             match beacon_verify_result.result {
                 VerificationStatus::Valid => {
                     // beacon is valid, verify the witnesses
-                    let beacon_info: FollowerGatewayResp =
-                        beacon_verify_result.gateway_info.unwrap();
+                    let beacon_info: GatewayInfo = beacon_verify_result.gateway_info.unwrap();
```

Comment:
ahhhhhh unwrap ? No worries, i have refactored this out on the tidy up PR but we should get this PR in first and then i will resolve the conflicts on the tidy up pr.